scout/parse/variant/variant.py

```diff
@@ -418,6 +424,93 @@ def set_str_info(variant: Variant, parsed_variant: Dict[str, Any]):
     parsed_variant["str_inheritance_mode"] = call_safe(str, variant.INFO.get("InheritanceMode"))
 
 
+def set_fusion_info(variant: Variant, parsed_variant: Dict[str, Any]):
+    """Add Fusion information if present."""
+
+    def replace_nan(value: str, nan_value: str = "nan", replace_by: Any = "") -> str:
```

Comment:
You might apply the function also to the orientantion:

<img width="279" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/e2475661-5739-4cab-8f94-e9d9432cbf6f">